bin/compile_dependencies.sh

```diff
@@ -6,15 +6,21 @@ toplevel=$(git rev-parse --show-toplevel)
 
```

Comment:
Zou je boven dit bestand nog het stukje documentatie willen toevoegen? Zoals [bijvoorbeeld in ZTC](https://github.com/VNG-Realisatie/catalogi-api/blob/master/bin/compile_dependencies.sh#L4).